src/Biomass/FarmValues/components/FarmValuesTable/FarmValuesTable.js

```diff
@@ -14,17 +13,11 @@ import {
   FormControl,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
-import MuiAlert from '@material-ui/lab/Alert';
-import IssueDialogue from '../../../../Comments/components/IssueDialogue/IssueDialogue';
+import IssueDialogue from '../../../../Comments/components/IssueDialogue/IssueDialogue.js';
```

Comment:
there shouldn't be a .js here

Reply:
sorry, a bad habit from when I code on my own. I always think it makes it more clear to the compiler but obviously that's not how it works